README.md

```diff
@@ -0,0 +1,181 @@
+# GitHub Actions CI workflows, actions for Vald client
+
+A repository of workflows and actions for Vald client CI configuration.
+
+- [vald-client-go](https://github.com/vdaas/vald-client-go)
+- [vald-client-node](https://github.com/vdaas/vald-client-node)
+- [vald-client-python](https://github.com/vdaas/vald-client-python)
+- [vald-client-java](https://github.com/vdaas/vald-client-java)
+
+## The workflows/actions
+
+### Workflows
+
+- [Sync](./.github/workflows/sync.yaml)
+  - This workflow is designed to handle synchronization with the Vald repository.
+  - It checks for differences between the `main` branch of the Vald client repository and the `main` branch of the Vald repository, and synchronizes API changes.
+  - Additionally, it synchronizes release tags if a release occurs in the Vald repository.
+  - This automation helps maintain alignment and reduces manual effort in keeping repositories synchronized.
+
+- [Release](./.github/workflows/release.yaml)
+  - This workflow is designed to handle the release process, including  packaging the Vald client for distribution.
+  - It also handles the publication to external package repositories, ensuring that the released versions are available to the community.
+    - For Python projects, the release is published to [PyPi](https://pypi.org/project/vald-client-python/), allowing Python users to install via `pip`.
+    - For Node.js projects, you can find the release on [npm](https://www.npmjs.com/package/vald-client-node) and install it using `npm install`.
+    - For Java projects, Maven artifacts are available [here](https://central.sonatype.com/artifact/org.vdaas.vald/vald-client-java) for integration into projects.
```

Comment:
How about Clojure projects?

Reply:
Thank you for catching it!! I forgot to add it 🙏 
I'll add it 👍

Reply:
https://github.com/vdaas/vald-client-ci/pull/5/commits/7d8b64499cb27333e0c7458fe206e7292346b0c2